README.md

```diff
@@ -48,3 +48,30 @@ Or do `GET /users.csv`.
 The header approach is prefered.
 
 Currently the `content-disposition` header is set to `attachment;` by default since this plugin is intended for exporting purposes, if this hinders you just let us know.
+
+To handle typical pagination responses like
```

Comment:
I might be worth noting that this will apply to all routes/export cases.
Also we might want to add a section to explain all options but that is not for the scope of this PR :P